docs/source/technical_features.md

```diff
@@ -1,13 +1,36 @@
 # Technical features
 
-## Physic-informed network
+* [Physic informed network](#physic-informed-network)
```

Comment:
i don't think a "manual" TOC is necessary because a TOC will automatically be generated on the right (RTD theme)-- the problem of a manual TOC is maintenance

Reply:
Agree ++, removed!